meta-luneos/recipes-webos/nyx-modules/nyx-modules/0008-Add-LuneOS-modules-and-machine-specific-cmake-file-t.patch

```diff
@@ -79,7 +79,3 @@ index d435c35..1aed92a 100644
 +if(NYXMOD_OW_HAPTICS)
 +    add_subdirectory(haptics_timedoutput)
 +endif()
-+
-+if(NYXMOD_OW_GPS)
-+    add_subdirectory(gps)
-+endif()
```

Comment:
Not sure to understand this change here, don't we want the gps nyx module if we request it in the cmake file?

Reply:
It's duplicated (see above, already there). I added it by mistake when rebasing OSE, however since we didn't include NYXMOD_OW_GPS in any of the CMake files it never showed up during build until now.